iptables/iptables.go

```diff
@@ -61,19 +61,18 @@ type FirewallConfiguration struct {
 // https://github.com/istio/istio/blob/e83411e/pilot/docker/prepare_proxy.sh
 func ConfigureFirewall(firewallConfiguration FirewallConfiguration) error {
 
-	log.Printf("Tracing this script execution as [%s]\n", ExecutionTraceID)
+	log.Debugf("Tracing this script execution as [%s]", ExecutionTraceID)
 
-	startSection("current state")
+	log.Debugln("current state")
 	b := bytes.Buffer{}
 	if err := executeCommand(firewallConfiguration, makeShowAllRules(), &b); err != nil {
-		log.Println("Aborting firewall configuration")
+		log.Error("Aborting firewall configuration")
 		return err
 	}
-	endSection()
 
 	commands := make([]*exec.Cmd, 0)
 
-	startSection("configuration")
+	log.Debugln("configuration")
 
 	matches := chainRegex.FindAllString(b.String(), 1)
 	if len(matches) > 0 {
```

Comment:
Perhaps if we remove the `configuration` statement from above, we can expand a little bit here, something like:
`log.Printf("skipping iptables setup, found %d existing chains: %v", len(matches), matches)` 

or something similar to this

Reply:
Ok @mateiidavid , I'm adding it on my next commit. Followup question, though, do you guys think that the chains themselves are worthy to be logged as Info level? I thought maybe putting it aside to debug one.

Reply:
Yeah think that's a sensible choice.